app/views/compute_resources_vms/form/vmware/_volume.html.erb

```diff
@@ -23,3 +23,17 @@
            :label => _("Eager zero"), :label_size => "col-md-2", :disabled => !new_host},
            "true",
            "false" %>
+<%
+# regarding to the following documentation:
+# http://pubs.vmware.com/vsphere-65/index.jsp?topic=%2Fcom.vmware.wssdk.apiref.doc%2Fvim.vm.device.VirtualDiskOption.DiskMode.html
+# there are some more disk modes available (nonpersistent, undoable, append) but as far as I can see, 
+# these disk modes don't work and don't exist on current vmware vsphere configuration GUI.
+#  ['nonpersistent', :nonpersistent], 
+#  ['undoable', :undoable],
+#  ['append', :append]]
+%>
+<% disk_mode_values = [['Dependent', :persistent],
```

Comment:
Please move these options to vmware's compute resource model.

`firmware_types` is one example for this.
https://github.com/theforeman/foreman/blob/develop/app/models/compute_resources/foreman/model/vmware.rb#L170

Please note, that you need to mark the names as translatable, e.g. `_('Dependent')`.